### PR TITLE
provides better resources for end users

### PR DESCRIPTION
**Pull request by zerodouglas**
This PR helps provide a small insight on building the `phonon-client` from source, adds issue templates and other such information that may be helpful.

### CODE REVIEW

**Comment by zerodouglas** [comment]
This should resolve #217 .

**Comment by zerodouglas** [comment]
It may be better to list the OS field as a text field, allowing the end user to provide the OS's version, name, and other relevant information.

File: README.md [hunk 1]
```diff
@@ -41,35 +51,49 @@ go run main/phonon.go init
...
+
 ```
 export PATH=$PATH:`go env GOPATH`/bin
 ```
+
 It's recommended to set this variable in your shell profile to make this value persist in new sessions.
 
 ## Process
+
+This repository has a submodule that contains the frontend application. Use the `--recursive` flag to clone all submodules in this repository. Otherwise, you may encounter build errors.
```
Comment (npmaile) [inline README.md:81]: no mention of `make checkout-submodules` target

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    id: reproducible_steps
+    attributes:
+      label: Steps to reproduce
+  - type: dropdown
+    id: os
+    attributes:
+      label: OS
+      multiple: true
+      description: What operating system does this bug occur on?
+      options:
```
Comment (npmaile) [inline .github/ISSUE_TEMPLATE/bug-report.yml:24]: also need the version. should be a text box

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        - Windows
+        - macOS
+        - Linux
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: The browser(s) that this problem occurs in
+      multiple: true
+      options:
```
Comment (npmaile) [inline .github/ISSUE_TEMPLATE/bug-report.yml:33]: should also be a text box.

**Review by npmaile — CHANGES_REQUESTED** [review]

**Comment by zerodouglas** [comment]
Removes paragraph in README, due to Nate's change.

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 2 above)
Comment (zerodouglas) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: Changes can be viewed at https://github.com/GridPlus/phonon-client/pull/225/files#diff-7c49a365610d935ce439e3f241cd3f5d1df89a95d1a674d11ebdb1e518c7335cR18

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 3 above)
Comment (zerodouglas) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: Change at https://github.com/GridPlus/phonon-client/pull/225/files#diff-7c49a365610d935ce439e3f241cd3f5d1df89a95d1a674d11ebdb1e518c7335cR23

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+First of all, contributions are all very welcome. If you're unsure or afraid of anything, please submit an issue or pull request anyways. We're all very friendly and we deeply appreciate your interests and best efforts.
+
+## Issues
+
+Ensure that you've used the latest version before submitting a bug report. We may have already provided a fix for the issue.
+
+We've provided an issue template that helps set the guidelines on what we expect to help provide us with enough information to understand the problem and provide a solution.
+
+We may ask questions and so we ask that you respond quickly as possible to your issue.
```
Comment (maevyn11) [inline CONTRIBUTING.md:11]: should be "respond as quickly as"

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+## Pull requests
+
+Before creating a pull request, we ask that you create an issue for it or atleast reference an existing issue. This will help prevent duplication and overlapping in WIP work, and to allow us to provide more guidance.
+
+Your pull request should be descriptive, containing how it works and what has changed.
+
+Changes may be requested in your pull request. We ask that you do not use the resolve prompt to resolve your own issues. We will use this prompt once we can verify that the changes are made.
```
Comment (maevyn11) [inline CONTRIBUTING.md:19]: Would include a little more info here about the PR process. Like, merging requires that at least one phonon core developer approve the change, PRs must have no conflicts with master, and CI automated build must pass before changes will be merged.

**Review by maevyn11 — CHANGES_REQUESTED** [review]

**Review by maevyn11 — APPROVED** [review]
Leaving approval so this isn't held up while I'm out.

**Review by npmaile — APPROVED** [review]
el gee tee em

**Pull request merged**